runtime/sema/check_composite_declaration.go

```diff
@@ -1879,24 +1955,27 @@ func (checker *Checker) defaultMembersAndOrigins(
 
 		identifier := function.Identifier.Identifier
 
-		functionType := checker.functionType(function.Purity, function.ParameterList, function.ReturnTypeAnnotation)
+		functionAccess := checker.accessFromAstAccess(function.Access)
+
+		functionType := checker.functionType(function.Purity, functionAccess, function.ParameterList, function.ReturnTypeAnnotation)
```

Comment:
Chopping down the argument list makes it easier to read

```suggestion
		functionType := checker.functionType(
			function.Purity, 
			functionAccess, 
			function.ParameterList, 
			function.ReturnTypeAnnotation,
		)
```